### PR TITLE
Defer browser support documentation to the actual FT document.

### PR DESCRIPTION
**Pull request by notlee**
This is better than maintaining an out of date copy.

### CODE REVIEW

File: _components/compatibility.md [hunk 1]
```diff
@@ -76,134 +76,7 @@ If there is JavaScript to execute regardless of the experience served to the bro
...
-			<td>—</td>
-		</tr>
-		<tr>
-			<td>Safari</td>
-			<td>Latest</td>
-			<td>—</td>
-		</tr>
-	</tbody>
-</table>
+Origami follow the <a href="https://docs.google.com/document/d/1z6kecy_o9qHYIznTmqQ-IJqre72jhfd0nVa4JMsS7Q4/" class="o-typography-link--external"><abbr title="Financial Times">FT</abbr> Browser Support Policy document</a> available to <abbr title="Financial Times">FT</abbr> staff.
```
Comment (JakeChampion) [inline _components/compatibility.md]: Is this document the official browser support document? How do we get notified when it's content is changed? Is that possible?

Comment (notlee) [reply _components/compatibility.md]: It's at least canonical (recent update, the old FT support document links there, has more detail than what's publicly listed on ft.com). I don't think there's a process for updating it regularly so am not confident in getting an update by a person -- unless there's a Google Docs feature

Comment (chee) [reply _components/compatibility.md]: Why is this not the official browser support document? https://help.ft.com/help/tools-services/browser-compatibility/what-browsers-does-the-ft-support/

Comment (notlee) [reply _components/compatibility.md]: >Our publicly stated support policy is available on help.ft.com. It is aligned with this document but has less detail.

It also doesn't mention specialist titles or internal products

**Pull request merged**